packages/token/src/index.ts

```diff
@@ -27,25 +32,39 @@ export const STABLECOINS = ["USDC", "USDT"];
 
 /** The token class */
 export class Token {
-    private static tokenMapping: TokenMapping;
+    private static tokenMappings: Partial<Record<ChainId | "default", TokenMapping>>;
```

Comment:
We should have a static variable DEFAULT_CHAIN that gets set to the first chain that is loaded. Then we don't need to do `ChainId | "default"` or populate the default key.

Reply:
The `"default"` key is intended to be temporary, we need it in order to deprecate the `parseRemapFromString` method - I can document this better